supply-chain-security/compromises/README.md

```diff
@@ -13,6 +13,7 @@ We welcome additions to this catalog by
 
 | Name              | Year               | Type of compromise    | Link        |
 | ----------------- | ------------------ | ------------------    | ----------- |
+| [purescript-npm](2019/purescript-npm.md) | 2019 | Source Code Compromise | [1](https://harry.garrood.me/blog/malicious-code-in-purescript-npm-installer/) |
```

Comment:
Is there a link that points to an organization blog or news page? Would be preferred to someone's personal blog.